canopy/canopyjs/src/index.ts

```diff
@@ -13,6 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+interface User {
+  userId: string;
+  displayName?: string;
```

Comment:
In the REST API side, currently there is no concept of a display name. I imagine that would be included when we have user preference's implemented? So, I don't know if it makes sense to include this here at this time.

Reply:
It could be part of a the client codes transform from the rest api to this structure.  Presumably the display name would be in some sort of arbitrary properties.

Reply:
So yes, `displayName` is an optional argument and it is not meant to be a 1-to-1 with the data that is exposed via the API. The structure of the user object on the front-end should be as flat as possible and this is meant to illustrate that. Perhaps a comment would be nice, but I don't see it as a necessity.